packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/WholeMeetingSummary.tsx

```diff
@@ -47,7 +47,7 @@ const WholeMeetingSummary = (props: Props) => {
     const reflections = reflectionGroups?.flatMap((group) => group.reflections) // reflectionCount hasn't been calculated yet so check reflections length
     const hasMoreThanOneReflection = reflections?.length && reflections.length > 1
     if (!hasMoreThanOneReflection || !organization.useAI || !hasAiApiKey) return null
-    if (!wholeMeetingSummary) return <WholeMeetingSummaryLoading />
+    if (wholeMeetingSummary === '') return <WholeMeetingSummaryLoading /> // summary is undefined if it hasn't been generated yet. it's null if unsuccessful
```

Comment:
-1 GraphQL does not distinguish between `null` and `undefined` like this, so while it does not change anything here, the comment is wrong.
I think because the summary is only used for the AI summary, it's ok to have a placeholder in the db indicating loading, but I would prefer if on GraphQL level we would add a field `isSummaryLoading` which checks the placeholder. We could also filter it out for the `summary` field. This way it's contained to the server only. Not sure `''` is the best placeholder though, maybe something like `<loading>` would be more explicit?